src/cmdSwitches.js

```diff
@@ -1,9 +1,9 @@
 const { app } = require('electron');
 
 const presets = {
-  'base': '--autoplay-policy=no-user-gesture-required --disable-features=WinRetrieveSuggestionsOnlyOnDemand,HardwareMediaKeyHandling,MediaSessionService', // Base Discord
```

Comment:
Please don't modify `base`, it is base Discord's arguments, not base OpenAsar (see the comment).